Fix watch star issues

yougane opened the pull request.

yougane committed:
Fix watch button

yougane committed:
Use filled star instead of outlined star when activated

dgaedcke reviewed (changes requested):
I'm noticing the "WatchButton" widget defined in two separate files.  Is that intentional??

Also, can you remove all these build-artifact files from the repo and the PR.  I'm not sure how it happened, but we should not have git watching them

yougane committed:
Add gitignore to remove build artifacts from the repo

yougane committed:
Delete build artifacts from the repo

yougane committed:
Remove .packages from the repo

yougane committed:
Remove pubspec.lock from the repo

yougane commented:
@dgaedcke I removed the build artifacts from the repo and added a .gitignore to prevent these files from being pushed to git again.

The WatchButton widget is defined only in [shared_row_components.dart](https://github.com/NewMillennialGames/st_event_config_cli/pull/6/files#diff-7244bc8300cde1e40e8343cdeeabdeb2cfcd64c31968294b125a7bc6f7164a89)
